src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -122,12 +126,21 @@ class NUnit extends ConventionTask {
         project.file(getReportFolder())
     }
 
+    File getLogFolderImpl() {
+        if (logFolder)
+            project.file(logFolder)
+        else
+            return null
+    }
+
     @OutputFile
     File getTestReportPath() {
         // for the non-default nunit tasks, ensure we write the report in a separate file
-        // TODO: Do we need to supply prefix when user has specified its own report file name?
-        def reportFileNamePrefix = name == 'nunit' ? '' : name
```

Comment:
There is no possibility to specify name without prefix. But user still can set the same report file in own tasks.